apps/web-site/src/app/App.vue

```diff
@@ -172,9 +172,6 @@ body {
 p {
   margin: 0;
 }
-.pm-p {
-  min-height: 18px;
-}
```

Comment:
Hi @sampullman 

I'm not sure if I'm getting this wrong; I think the prosemirror editor is only visible in the builder, but it seems that currently the styles of `.pm-p` is defined in `apps/web-site/src/app/App.vue` and `pubstudio-builder/libs/frontend/feature-build/src/components/preview/SitePreview.vue`.

I moved the style definition of `.pm-p` to `apps/web/src/app/App.vue` instead so that only the builder app is affected. A new [PR](https://github.com/samatechtw/pubstudio/pull/927) is created in the pubstudio repo for this purpose.

Reply:
This should be left in, otherwise a double line break will not show the correct spacing in the live version. `.pm-p` is the class added to each div generated by ProseMirror. You are correct that we need it in the `pubstudio` repo as well, though.

However, I realized an obvious mistake in my implementation - it doesn't account for different line heights! Below is the live version (without `pm-p` style), editor version (with `pm-p` style), and the editor version with selected component.

![space](https://github.com/samatechtw/pubstudio-builder/assets/550982/2de69efc-51c4-4385-a070-2b550fab6cf8)

I think I've found a solution though, we can use a zero width character to the editor/live style:
```css
.pm-p:after {
  content: '\200b';
}
```

This will cause the selected/active ProseMirror editor to have line breaks that are double height, but it can be solved by setting the ProseMirror editor line break to `display: none;`. Hopefully I didn't miss anything 🤞